src/api/app/assets/stylesheets/webui/notification_toolbar.scss

```diff
@@ -0,0 +1,8 @@
+.notification-toolbar.sticky-top {
+  z-index: calc(#{$zindex-modal} - 10);
```

Comment:
@OleksandrOrlov I think you need to check the z-index again, it overlaps with the watchlist on mobile 
![Screenshot_2020-09-29 Request 8 (declined)](https://user-images.githubusercontent.com/22001671/94557981-a054e900-025f-11eb-8e8c-914563d031a6.png)

Reply:
Fixed.